README.md

```diff
@@ -1,28 +1,18 @@
 # Promise Katas
-This material accompanies the **Working with asynchronous Javascript** lecture.
 
-## How to set up
+This repository contains a set of functions I wrote with Express to practise fetching data from a server and returning promises.
 
-Clone and run `npm install`
+The starter files and end-to-end tests were cloned from: https://github.com/CommandShiftHQ/be-async-promise-katas
 
-To run the tests run `npm t` or `npm test`
+## Getting started
 
-## Instructions
+- Create a fork of this repo.
+- Clone your copy of the repo using the command: git clone git@github.com:_your-github-username_/javascript-basics-express
```

Comment:
You dont have to be overly specific with cloning instructions, just saying `git clone` here is fine
```
- Copy the forks git address clone it to your machine with `git clone`
```